src/components/Profile.js

```diff
@@ -21,7 +21,8 @@ class Profile extends React.Component { // eslint-disable-line react/prefer-stat
         return (
             <UncontrolledDropdown nav inNavbar>
                 <DropdownToggle nav caret>
```

Comment:
You will need to add `className="navbar-user"` as a prop here to get the user dropdown styling.